### PR TITLE
Add quiet mode.

### PR DESCRIPTION
**Pull request by zoilomora**
- Add quiet mode.
- Add .idea to .gitignore.

### CODE REVIEW

**Comment by zoilomora** [comment]
The travis joblog shows this error: ``The command "vendor/bin/test-reporter" exited with 1.``

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 composer.lock
 vendor
 build
+.idea
```
Comment (thiagoalessio) [inline .gitignore]: this has nothing to do with the pull request you proposed, and on top of that it isn't nice to pollute someone's `.gitignore` with files that are irrelevant to them. (abandon IDEs, come to Vim 🤣 )

**Comment by thiagoalessio** [comment]
* one more thing ... I fixed the issue that was failing the travis build, feel free to rebase master and push again ;D

File: .gitignore (hunk 1 above)
Comment (zoilomora) [inline .gitignore]: PhpStorm Forever!! XD

**Comment by zoilomora** [comment]
everything is correct? It's my first pull request 😅

**Comment by thiagoalessio** [comment]
all good now, thank you very much for your contribution ;D
👍👍👍👍

**Comment by zoilomora** [comment]
you are welcome! 😁

**Pull request merged**